packages/suite/src/views/wallet/sign-verify/index.tsx

```diff
@@ -49,20 +46,62 @@ const Form = styled.form`
     }
 `;
 
-const SignVerify = () => {
-    const [page, setPage] = useState<NavPages>('sign');
+const FormatDescription = styled.p`
+    span {
+        font-weight: ${variables.FONT_WEIGHT.BOLD};
+    }
+
+    & + & {
+        margin-top: 10px;
+    }
+`;
 
+const StyledSelectBar = styled(SelectBar)`
+    margin: 12px 0 20px;
+
+    @media (min-width: ${variables.SCREEN_SIZE.SM}) {
+        width: 320px;
+        margin: 0 0 0 20px;
+    }
+
+    @media (min-width: ${variables.SCREEN_SIZE.MD}) and (max-width: ${variables.SCREEN_SIZE.LG}) {
+        width: 100%;
+        margin: 12px 0 20px;
+    }
+
+    @media (min-width: ${variables.SCREEN_SIZE.LG}) {
+        width: 320px;
+        margin: 0 0 0 20px;
+    }
+`;
+
+const Divider = styled.div`
+    margin: 15px 0 30px;
+    height: 1px;
+    background: ${({ theme }) => theme.STROKE_GREY};
+`;
+
+const Copybutton = styled(Button)`
+    position: absolute;
+    right: 0;
+    top: -2px;
+`;
+
+const SignVerify: React.FC = () => {
     const { selectedAccount, revealedAddresses } = useSelector(state => ({
         selectedAccount: state.wallet.selectedAccount,
         revealedAddresses: state.wallet.receive,
     }));
 
-    const { isLocked } = useDevice();
-    const { translationString } = useTranslation();
+    const [page, setPage] = useState<NavPages>('sign');
+    const [isCompleted, setIsCompleted] = useState(false);
```

Comment:
Nitpick: I agree with the `isFoobar` prefix, but I'm not sure about `setIsFoobar` variant. I've already seen `[isFoobar, setFoobar]` and I like it a bit more even if it violates the `[X, setX]` convention. But I'm willing to change my mind.

Reply:
I am not a fan of violating the pattern but a fan on the prefix. Also, if public code snippets are used the setter would be named `setIs` anyways 🤷‍♂️

Reply:
Ok, let's leave it this way. :+1: